tsconfig.json

```diff
@@ -13,15 +13,11 @@
     "declaration": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
     "preserveWatchOutput": true,
   },
-  "exclude": [
-    "frontends/*/**/*.test.ts",
-    "frontends/*/**/*.test.tsx"
-  ]
```

Comment:
Originally, we compiled TS to JS use `tsc` (the typescript compiler) and stored the JS files on disk prior to building the app via webpack. We excluded test files intentionally since they did not need to be transpiled.

Now we do not actually transpile any TS to JS prior to building webpack, so I see no reason to exclude them from our normal tsconfig. Consequences:
- TS files are now typechecked
- test files use the same tsonfig as our main files; previously, I am not entirely sure what tsconfig it was using for test files. Possibly some default from `ts-node`.

    It's nice to use the same config for test+regular files because some settings (like `moduleResolution`) are important to have the same.


*Alternatively, we could have different configs for test & non-test files that extend the same base config; but since we are bundling the app, I don't think there's any reason to use two separate configs.*